<ui version="4.0" >
 <class>DCodeGenerationFormBase</class>
 <widget class="QWidget" name="DCodeGenerationFormBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>376</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item row="3" column="0" >
      <widget class="QGroupBox" name="groupBox3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="title" >
        <string>Project Generation</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="m_makeANTDocumentCheckBox" >
          <property name="text" >
           <string>Create ANT build document</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QGroupBox" name="groupBox2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Auto-Generate Methods</string>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="m_generateConstructors" >
            <property name="text" >
             <string>Empty constructor methods</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="m_generateAssocAccessors" >
            <property name="text" >
             <string>Association accessor methods</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="m_generateAttribAccessors" >
            <property name="text" >
             <string>Attribute accessor methods</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel1_3" >
            <property name="text" >
             <string>Default attribute accessor scope:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KComboBox" name="m_accessorScopeCB" >
            <property name="currentIndex" >
             <number>3</number>
            </property>
            <item>
             <property name="text" >
              <string>Public</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Private</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Protected</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>From Parent Object</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" >
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel1_3_2" >
            <property name="text" >
             <string>Default association field scope:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KComboBox" name="m_assocFieldScopeCB" >
            <property name="currentIndex" >
             <number>3</number>
            </property>
            <item>
             <property name="text" >
              <string>Public</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Private</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Protected</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>From Parent Role</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QGroupBox" name="groupBox1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="title" >
        <string>Documentation</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ui_textLabelStyle" >
          <property name="text" >
           <string>Style:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="m_SelectCommentStyle" >
          <item>
           <property name="text" >
            <string>Slash-Slash (//)</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Slash-Star (/**  */)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="ui_textLabelTitle" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;p align="center">D Code Generation&lt;/p></string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
